src/libpgmoneta/configuration.c

```diff
@@ -82,6 +82,9 @@ static int restart_string(char* name, char* e, char* n);
 static bool is_empty_string(char* s);
 static int remove_leading_whitespace_and_comments(char* s, char** trimmed_line);
 
+static void trim_spaces(char* str);
```

Comment:
`char* pgmoneta_remove_whitespace(char* orig);` ?

Reply:
`remove_leading_whitespace_and_comments` does not remove the trailing whitespace.

Reply:
I didn't know about this function, so thanks for reminding me. I changed to using `pgmoneta_remove_whitespace`. While testing, I encountered an error in `bzip2_compression.c`. If I am not mistaken, I believe there is a logic error in the current functions within `bzip2_compression.c`, such as in the `bzip2_compress` function. This function will always return 1. I found this error while working on PR #345. Since I am unfamiliar with bzip2, I added a new function to make the test successful.

Reply:
If you found a bug then you can create an issue for it, and submit it separately

Reply:
I don't know if it is a bug or if I was wrong.